nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -270,8 +279,9 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
                                         data.put("columnQualifier", key.getColumnQualifier().toString());
                                         data.put("columnVisibility", key.getColumnVisibility().toString());
                                         data.put("timestamp", key.getTimestamp());
+                                        data.put("value", kv.getValue());
```

Comment:
As mentioned in the RecordField definition, `Value.toString()` converts the underlying byte array to a UTF-8 encoded String.  If that covers all use cases, this should be changed to `kv.getValue().toString()`.  Alternatively, if the underlying byte array should be represented, this should be changed to `kv.getValue().get()` and the RecordFieldType should be changed,